### PR TITLE
added changes specifically for deploying to Azure app servivce or con…

### PR DESCRIPTION
**Pull request by katherinewu007**
…tainer app

### CODE REVIEW

File: core/downloader.py [hunk 1]
```diff
@@ -40,6 +40,7 @@ def download(id, name=""):
         return save_name
     except Exception as e:
         core.updatelog("Error in downloader.py")
+        core.updatrlog(e)
```
Comment (boppitybop) [inline core/downloader.py]: typo?

File: static/js/main.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ var elements = $('.modal-overlay, .modal');
 $('.close-modal').click(function() {
     elements.removeClass('active');
 });
-var csrftoken = $('meta[name=csrf-token]').attr('content')
+var csrftoken = "lol"
```
Comment (boppitybop) [inline static/js/main.js]: what's this for?

File: core/downloader.py (hunk 1 above)
Comment (katherinewu007) [reply core/downloader.py]: Ah yes, I'll update

File: static/js/main.js (hunk 2 above)
Comment (katherinewu007) [reply static/js/main.js]: To remove the csrf token validation to be able to use the api (originally they only allow the UI use), can remove everything relating to it actually.

**Review by boppitybop — APPROVED** [review]

**Pull request merged**